org.eclipse.scout.rt/pom.xml

```diff
@@ -158,9 +158,14 @@
     <dependencies>
       <!-- Build Dependencies -->
       <dependency>
-        <groupId>com.sun.mail</groupId>
-        <artifactId>jakarta.mail</artifactId>
-        <version>2.0.1</version>
+        <groupId>jakarta.mail</groupId>
+        <artifactId>jakarta.mail-api</artifactId>
+        <version>2.1.3</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.angus</groupId>
+        <artifactId>angus-mail</artifactId>
```

Comment:
Does it include the API? Do we need an additional API dependency (before this one)?